### ISSUE TITLE
StarRating - doesn't have focus indicator

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7452]
### Expected Behavior
The StarRating component should have a focus indicator when navigating by keyboard

### Actual Behavior
No focus indicator is shown

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/input-starrating-simple--simple

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):

**Comment by jcfilben** [comment]
Resolved by #7466
